Markdown/Markdown.sublime-syntax

```diff
@@ -411,42 +411,36 @@ contexts:
   escape:
     - match: '{{escape}}'
       scope: constant.character.escape.markdown
+  atx-heading-terminator:
+    - match: '[ ]*(#*)[ ]*(?=$)($\n?)' # \n is optional so ## is matched as end punctuation in new document (at eof)
```

Comment:
The `(?=$)` lookahead is redundant here.

Reply:
Good catch, thanks. Leftover from earlier iterations.